Mathlib/Analysis/BoxIntegral/Partition/Basic.lean

```diff
@@ -142,10 +142,8 @@ instance partialOrder : PartialOrder (Prepartition I) where
       fun π₁ π₂ h₁ h₂ => injective_boxes (Subset.antisymm (this h₁ h₂) (this h₂ h₁))
     intro π₁ π₂ h₁ h₂ J hJ
     rcases h₁ hJ with ⟨J', hJ', hle⟩; rcases h₂ hJ' with ⟨J'', hJ'', hle'⟩
-    obtain rfl : J = J''
-    · exact π₁.eq_of_le hJ hJ'' (hle.trans hle')
-    obtain rfl : J' = J
-    · exact le_antisymm ‹_› ‹_›
+    obtain rfl : J = J'' := π₁.eq_of_le hJ hJ'' (hle.trans hle')
```

Comment:
Nice find. (At some point, I'd like to make the subgoal form of obtain warn in mathlib, at least to fix all such cases...)